Frontend/tailored-brand-poc/src/app/invoice-detail/invoice-detail.component.ts

```diff
@@ -88,9 +88,9 @@ export class InvoiceDetailComponent implements OnInit {
   }
   
   getAllOrderNo() {
-    this.purchaseOrderService.list()
+    this.purchaseOrderService.list(0,false)
```

Comment:
start using named params from ES6, use destructuring it will help anyone understand the params

Reply:
@himanshu1365 ??